framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2311,6 +2311,17 @@ public boolean shouldSuppressWarnings(@Nullable TreePath path, String errKey) {
                     // elements, because they may not have an @AnnotatedFor.
```

Comment:
Can you also think about what these comments mean and possibly clarify them?

Reply:
Update the comment in latest commit.